docs/wiki/frameworks/god-mode.md

```diff
@@ -0,0 +1,80 @@
+---
+title: God Mode
+---
+
+{% include important.html content="Dev-build only" %}
+
+All [God Mode](/wiki/user/god-mode) configuration only applies to the local machine. This allows great flexibility in configuration and different group presets for each player.
+
+## Access
+
+Current Chat Channel and Groups have separate access rights.
+
+```js
```

Comment:
Other parts of the docu are formatted using c++ iirc

Reply:
Nope, it should use `js` because it fits better for SQF. C++ is used for config.